subsys/bluetooth/services/fast_pair/include/fp_crypto.h

```diff
@@ -7,6 +7,8 @@
 #ifndef _FP_CRYPTO_H_
 #define _FP_CRYPTO_H_
 
+#include <fp_storage.h>
```

Comment:
This crypto module dependency on the Storage module looks strange. Maybe we should have a common header for definitions required by multiple modules e.g fp_internal.h